client/my-sites/stats/stats-notices/do-you-love-jetpack-stats-notice.tsx

```diff
@@ -102,7 +102,7 @@ const DoYouLoveJetpackStatsNotice = ( {
 		? translate(
 				'Finesse your scaling-up strategy with detailed insights and data. Upgrade to an Explorer plan for a richer understanding and smarter decision-making.'
 		  )
-		: translate( 'Upgrade to get priority support and access to upcoming advanced features.' );
+		: translate( 'Upgrade to get priority support and stop the upgrade notices.' );
```

Comment:
I'm wondering about this line now. It seems priorities are changing in JPOP now so it might be better to go with something like, "Upgrade to support future development".

Ref: pbg9X-jyM-p2

Reply:
Hi @a8ck3n, can I lean on you to polish the PR to completion please? Thanks heaps 👍

Reply:
Sorry I missed this one. I think that was during my AFK earlier this week. Shipped changes look good though. 👍